org_fedora_oscap/rule_handling.py

```diff
@@ -668,3 +683,72 @@ def eval_rules(self, ksdata, storage, report_only=False):
             return []
 
     # nothing to be reverted for now
+
+
+class KdumpRules(RuleHandler):
+    """Simple class holding data from the rules affecting the kdump addon."""
+
+    def __init__(self):
+        """Constructor setting the initial value of attributes."""
+
+        self._kdump_enabled = None
+        self._kdump_default_enabled = None
+
+    def kdump_enabled(self, kdenabled):
```

Comment:
This method doesn't seem to be doing anything. Or am I missing something?

Reply:
@matejak logic in the method was switched and incorrect. I corrected it now.